prime-dotnet-webapi/Services/GisService.cs

```diff
@@ -83,10 +85,17 @@ public async Task<int> CreateOrUpdateGisEnrolmentAsync(GisChangeModel changeMode
             return currentGisEnrolment.Id;
         }
 
-        public async Task<bool> LdapLogin(string username, string password, ClaimsPrincipal user)
+        public async Task<LoginStatus> LdapLogin(string username, string password, ClaimsPrincipal user)
         {
-            var gisUserRole = await _ldapClient.GetUserAsync(username, password);
-            var success = gisUserRole == "GISUSER";
+            var ldapResponse = await _ldapClient.GetUserAsync(username, password);
+            var success = ldapResponse.GisUserRole == "GISUSER";
```

Comment:
Now that we are looking at the LDAP API more thoroughly, is `UserRole == "GISUSER"` really all that is required for the login to be successful? the LDAP API returns an `Authenticated` property, should we be checking that?